iroh-net/src/magicsock.rs

```diff
@@ -1869,15 +1826,6 @@ impl Actor {
         inc!(MagicsockMetrics, update_endpoints);
 
         debug!("starting endpoint update ({})", why);
-        if self.no_v4_send && !self.inner.is_closed() {
```

Comment:
should we error out here? seems in this case we are quite dead in the water

Reply:
I'm not sure there's anything for us to do really.  Trying to send and receive data will result in an error at which point the application code has to give up on the magicsock like it would have with a normal socket.

The other option is to short-cut this into a fatal error that's propagated in `.poll_send()` or `.poll_recv()`, but it achieves just the same.